apps/web/components/EscrowTable/EscrowTable.tsx

```
@@ -0,0 +1,229 @@
+          Status: {dbStatus}
+        </p>
+        {error && <p className="text-red-500">Error: {error}</p>}
+      </div>
+
+      <div className="mb-4">
+        <h2 className="text-xl mb-2">Test Actions</h2>
+        <div className=" flex flex-col gap-3 lg:flex-row lg:gap-4 md:w-[40%]">
+          <button
+            onClick={insertTestData}
+            className="bg-blue-500 text-white px-4 py-2 rounded hover:bg-blue-600"
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)